src/openrct2/Game.cpp

```diff
@@ -845,3 +845,44 @@ void game_load_or_quit_no_save_prompt()
             break;
     }
 }
+
+void start_silent_record()
+{
+    char* name = tmpnam(nullptr);
+    auto* replayManager = OpenRCT2::GetContext()->GetReplayManager();
+    if (replayManager->StartRecording(name, OpenRCT2::k_MaxReplayTicks, OpenRCT2::IReplayManager::RecordType::SILENT))
+    {
+        OpenRCT2::ReplayRecordInfo info;
+        replayManager->GetCurrentReplayInfo(info);
+        safe_strcpy(gSilentRecordingName, info.FilePath.c_str(), MAX_PATH);
+
+        const char* logFmt = "Silent replay recording started: (%s) %s";
+        log_info(logFmt, info.Name.c_str(), info.FilePath.c_str());
+    }
+}
+
+bool stop_silent_record()
+{
+    auto* replayManager = OpenRCT2::GetContext()->GetReplayManager();
+    if (!replayManager->IsRecording() && !replayManager->IsNormalising())
+    {
+        return false;
+    }
+
+    OpenRCT2::ReplayRecordInfo info;
+    replayManager->GetCurrentReplayInfo(info);
+
+    if (replayManager->StopRecording())
+    {
+        const char* logFmt = "Replay recording stopped: (%s) %s\n"
+                             "  Ticks: %u\n"
+                             "  Commands: %u\n"
+                             "  Checksums: %u";
+
+        log_info(logFmt, info.Name.c_str(), info.FilePath.c_str(), info.Ticks, info.NumCommands, info.NumChecksums);
```

Comment:
This could get annoying to see in the console, I would prefer `std::printf` to be used, just so you don't constantly see the source filename and line number.

Reply:
done